### PR TITLE
fix(website): Fix headings in elevator and feature cards

### PR DESCRIPTION
**Pull request by Patticatti**

### CODE REVIEW

File: website/src/components/ElevatorPitch/index.tsx [hunk 1]
```diff
@@ -70,15 +70,15 @@ export default function ElevatorPitch() {
...
+    <div className="flex w-full h-fit lg:flex-row flex-col justify-center gap-6 lg:gap-12 xl:gap-20 items-center max-w-screen-xl ">
       <div className="flex flex-col w-full h-full justify-between lg:w-[480px] xl:w-[580px]">
         <div>
           <h6 className="uppercase text-sm font-semibold text-primary-450 tracking-wide mb-2">
             Stay Connected
           </h6>
           <div className="mb-2 text-3xl md:text-4xl lg:text-5xl ">
             <h3
-              className={`leading-tight text-pretty text-left tracking-tight font-bold inline-block ${manrope.className}`}
+              className={`leading-none text-pretty text-left tracking-tight font-bold inline-block ${manrope.className}`}
```
Comment (jamilbk) [inline website/src/components/ElevatorPitch/index.tsx]: I added `leading-tight` so the text wouldn't touch here. I think that looks better?

File: website/src/components/FeatureCards/index.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default function FeatureCards() {
           </h6>
           <div className="mb-4 lg:mb-0 text-3xl md:text-4xl lg:text-5xl">
             <h3
-              className={` text-pretty text-left leading-tight tracking-tight font-bold inline-block ${manrope.className}`}
+              className={` text-pretty text-left leading-none tracking-tight font-bold inline-block ${manrope.className}`}
```
Comment (jamilbk) [inline website/src/components/FeatureCards/index.tsx]: Same re: leading-tight

File: website/src/components/ElevatorPitch/index.tsx (hunk 1 above)
Comment (Patticatti) [reply website/src/components/ElevatorPitch/index.tsx]: hmm maybe we can get fb from the team?

File: website/src/components/FeatureCards/index.tsx (hunk 2 above)
Comment (Patticatti) [reply website/src/components/FeatureCards/index.tsx]: we can remove these changes - the important once is the one where i set the max width & gap so the widths of sections are always consistent

File: website/src/components/ElevatorPitch/index.tsx (hunk 1 above)
Comment (jamilbk) [inline website/src/components/ElevatorPitch/index.tsx]:
```suggestion
              className={`leading-tight text-pretty text-left tracking-tight font-bold inline-block ${manrope.className}`}
```

File: website/src/components/FeatureCards/index.tsx (hunk 2 above)
Comment (jamilbk) [inline website/src/components/FeatureCards/index.tsx]:
```suggestion
              className={` text-pretty text-left leading-tight tracking-tight font-bold inline-block ${manrope.className}`}
```

**Review by jamilbk — APPROVED** [review]

**Pull request merged**